system/ramspeed/ramspeed.c

```diff
@@ -0,0 +1,435 @@
+/****************************************************************************
+ * apps/system/ramspeed/ramspeed.c
```

Comment:
Can you please rename to `ramspeed_main.c`?

Reply:
Done